site/_js/slider.js

```diff
@@ -0,0 +1,183 @@
+import {gsap} from 'gsap';
```

Comment:
Please see my related comments about not pulling in `gasp` on every page.

Reply:
Done. Moved to /100/ directory.